neuraxle/steps/data.py

```diff
@@ -81,9 +80,8 @@ class EpochRepeater(MetaStepMixin, BaseStep):
     .. code-block:: python
 
         p = Pipeline([
-            TrainOnlyWrapper(DataShuffler(seed=42, increment_seed_after_each_fit=True)),
-            TrainOnlyWrapper(EpochRepeater(ForecastingPipeline(), epochs=EPOCHS))
-            TestOnlyWrapper(ForecastingPipeline())
+            TrainOnlyWrapper(DataShuffler(seed=42, increment_seed_after_each_fit=True, increment_seed_after_each_fit=False)),
+            EpochRepeater(ForecastingPipeline(), epochs=EPOCHS)
```

Comment:
This is what I wanted:

```
p = Pipeline([
    TrainOnlyWrapper(DataShuffler(seed=42, increment_seed_after_each_fit=True, increment_seed_after_each_fit=False)),
    TrainOnlyWrapper(EpochRepeater(epochs=EPOCHS)),
    ForecastingPipeline()
])
```

Otherwise, how is the EpochRepeater disabled during test time? 

EDIT: it's okay. Just ensure to add `repeat_in_test_mode=False` here (and above at the other place) in the doc so make it clear that it's disallowed or disallowable.